packages/orbit-components/src/ErrorForms.stories.tsx

```diff
@@ -376,28 +380,18 @@ export const RtlError = ({ label, error, value, placeholder, prefix, required })
         </InputGroup>
       </Stack>
     </RenderInRtl>
-  );
-};
-
-RtlError.story = {
-  name: "RTL error",
+  ),
 
   parameters: {
     info: "Some description about this type of InputField in general.",
+    controls: {
+      exclude: "help",
+    },
   },
 };
 
-RtlError.args = {
-  label: "Label",
-  error: "Something is not quite right",
-  value: "",
-  placeholder: "Placeholder",
-  prefix: "$",
-  required: true,
-};
-
-export const withModal = ({ label, error, showMore }) => {
-  return (
+export const WithModal: Story = {
+  render: ({ label, error, showMore }) => (
```

Comment:
Value, placeholder, prefix and required controls are not used here, let's disable them then? Or maybe I'm blind 😅